QUICK_START_GUIDE.md

```diff
@@ -11,175 +11,286 @@
 This Quick Start Guide will help you get InstructLab
 working on your laptop or machine and is expected to take approximately XX minutes. If you'd like more details on this process, see [the Taxonomy README](https://github.com/instructlab/taxonomy/blob/main/README.md) or if you'd like more information on the `cli`, please see [the ilab CLI README](https://github.com/instructlab/instructlab/blob/main/README.md)
 
-### Install `ilab`
-
-## 📋 Requirements
```

Comment:
If you are just receiving this document, you should see this first to see if you can even run this on your laptop.